app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def public_recipes
+    @recipes = Recipe.includes(:user).where(public: false)

```

Comment:
- [ ]  I wasn't able to tests that cause the recipe show not working but the next reviewer should review it